docker/setup.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
```

Comment:
it need to be separate because we need to use ```source activate beakerx``` which is only avaiable on /bin/bash shell